app/build.gradle

```diff
@@ -42,12 +49,14 @@ dependencies {
     // However, that is a different topic.
     def daggerVersion = '2.14'
     def butterKnifeVersion = '8.8.0'
+    def kotlinVersion = '1.2.0'
```

Comment:
There are newer versions of Kotlin (v1.2.20 as of writing this comment). I'm using *v1.2.0* here because it is the version that matches Android Studio 2.3.3 Kotlin plugin. 

Check here for the latest Kotlin releases: https://blog.jetbrains.com/kotlin/category/releases/.